misc/docker/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# This is a full install of Nit on a debian base.
+# Full because most dependencies are installed so that most tests can be run
+
+FROM debian:jessie
+MAINTAINER Jean Privat <jean@pryen.org>
+
+# Install dependencies
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+		# Recomanded builds pakages
+		build-essential \
+		ccache \
+		libgc-dev \
+		graphviz \
+		libunwind-dev \
+		pkg-config \
+		# Get the code!
+		git \
+		ca-certificates \
+		curl \
+		# For nit manpages :)
+		man \
+	&& rm -rf /var/lib/apt/lists/*
+
+# Clone and compile
+RUN git clone https://github.com/nitlang/nit.git /root/nit \
+	&& cd /root/nit \
+	&& make \
+	&& . misc/nit_env.sh install \
+	# Clean and reduce size
+	&& strip c_src/nitc bin/nit* \
+	&& ccache -C \
+	&& rm -rf .git
+
+# Dependencies for more libs and tests
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
```

Comment:
You should really install all the packages needed in one `apt-get install` run. Every `RUN` commands will create an intermediate docker image. In your case you don't really need it.